content/90.back-matter.md

```diff
@@ -2,3 +2,47 @@
 
 <!-- Explicitly insert bibliography here -->
 <div id="refs"></div>
+
+
+## Supplemental Tables
+
+| doi                   | PC_1              | figure                                                                                                                                                                    | 
```

Comment:
A few thoughts:
* This is an amazing table!
* Can we put the PC1 parts for low/high into the main text as a table?
* Can we add a column for the license?
* Can we include the figure directly in the table for those with a permissive license?